packages/fxa-settings/src/components/en.ftl

```diff
@@ -20,10 +20,14 @@
 # “Account” can be localized, “Firefox” must be treated as a brand.
 # This is used to refer to a user's account, e.g. "update your Firefox account ..."
 -product-firefox-account = Firefox account
-product-mozilla-vpn = Mozilla VPN
-product-pocket = Pocket
-product-firefox-monitor = Firefox Monitor
-product-firefox-relay = Firefox Relay
+-product-mozilla-vpn = Mozilla VPN
+-product-mozilla-mdn = Mozilla MDN
+-product-mozilla-hubs = Mozilla Hubs
+-product-pocket = Pocket
```

Comment:
`product-pocket` is also used in `pages/ResetPassword/CompleteResetPassword` and `pages/ResetPassword/ResetPassword`, and should be updated to `-product-pocket`.

Reply:
Ah thank you for catching this. Made the change.